src/styles/card.scss

```diff
@@ -0,0 +1,120 @@
+.product {
+  @include page-grid;
+
+  padding: 50px 40px;
+  justify-content: center;
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid #f0f1f4;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  flex-direction: column;
+  display: flex;
+  transition: transform 0.3s;
+
+  @include hover(transform, scale(1.2));
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__title {
+    font-weight: 500;
+    margin: 40px 0 4px;
+    line-height: 18px;
+    font-size: inherit;
+    color: $card-bolt-font-color;
+
+    &:hover {
+      cursor: pointer;
+      color: #34568b;
+    }
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__code {
+    color: $card-regular-font-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 0 16px;
+    height: 18px;
+  }
+
+  &__price-item {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $card-bolt-font-color;
+  }
+
+  &__price-name {
+    color: $card-regular-font-color;
+    line-height: 18px;
+  }
+
+  &__review-count {
+    color: $card-bolt-font-color;
+    font-size: 10px;
+    line-height: 14px;
+    height: 12px;
+  }
+
+  &__button {
+    width: 166px;
+    background-color: $blue-color;
+    color: #fff;
+    border-radius: 5px;
+    border: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    display: block;
+    text-align: center;
+    height: 40px;
+    margin: 0 auto;
+    outline: none;
+    padding: 12px;
+
+    &:hover {
+      background-color: #fff;
+      color: $blue-color;
+      border: 1px solid $blue-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+    margin-right: 4px;
+  }
+}
+
+.stars.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
```

Comment:
Move it to a separate block and create modifiers for each number of stars from 0 to 5